src/System.Linq.Parallel/src/System/Linq/Parallel/Utils/ExceptionAggregator.cs

```diff
@@ -116,13 +115,16 @@ internal static void ThrowOCEorAggregateException(Exception ex, CancellationStat
                 CancellationState.ThrowWithStandardMessageIfCanceled(
                     cancellationState.ExternalCancellationToken);
             }
+            else if (ex is AggregateException)
+            {
+                throw ((AggregateException)ex).Flatten();
+            }
```

Comment:
But this was by-design behavior, and it matches the desktop implementation, right?  We shouldn't be changing the product behavior as part of a set of test changes like this.